addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -110,4 +175,15 @@ def send_mail_test(self):
                 [Markup('<li>%s</li>') % notification_message for notification_message in notification_messages]
             ))
 
-        return True
+        if has_sent:
+            return {
+                'type': 'ir.actions.client',
+                'tag': 'display_notification',
+                'params': {
+                    'message': _('Test mailing successfully sent'),
```

Comment:
Technically not super correct, you could have sent to 3 mail addresses and only 1 worked.

We already log messages on the mass mailing record chatter, isn't it sufficient?